src/base/bittorrent/torrentimpl.h

```diff
@@ -51,16 +51,6 @@
 #include "torrent.h"
 #include "torrentinfo.h"
 
-#if (LIBTORRENT_VERSION_NUM == 20003)
```

Comment:
>This PR doesn't include anything about 2.0.4.

Here is it.
From now on, qBittorrent is incompatible with 2.0.3 (earlier versions of libtorrent 2 were unsupported before).

Reply:
oh, it's an implicit bump without configuration checks